docs/tutorials/gettingstartedexample/gse-gen-common.md

```diff
@@ -28,17 +28,267 @@ This document covers a range of common scenarios using the PXIe-148X Generation
+    - Select index <font face = "courier new">0</font> of the **Channel Configurations** array and make the following modifications.
+        - Select **Grayscale** on the **Interpretation** control in the **RAW Display Parameters** cluster.
+    - Select index <font face = "courier new">1</font> of the **Channel Configurations** array and make the following modifications.
+        - Set the **Serial Channel** control value to <font face = "courier new">SO1</font>.
+        - Select **Grayscale** on the **Interpretation** control in the **RAW Display Parameters** cluster.
+   
+   > The settings on the **Serial Channel** tab are now similar to the image below.
+   
+   ![Generating and Displaying Multiple Images Channel Configurations](../../images/PXIe-148X-Gen-MultipleImages-ChannelConfigurations.png)
+
+3. Run the VI, wait for the **Waiting for Serializer Setup** indicator to illuminate, and click the **Serializer Setup Complete** control button to start the generation.
+    > Once the generation completes, notice the **Number of Generated Packets** indicator array shows a value of <font face = "courier new">10820</font> at index 0 and <font face = "courier new">12820</font> at index 1, which equals the number of packets per frame (vertical resolution) times the number of frames generated for each channel. The indices in the **Number of Generated Packets** array correspond to the indices in the **Channel Configurations** array.
+
+    ![Generating and Displaying Multiple Images Generated Packets Indicator](../../images/PXIe-148X-Gen-MultipleImages-GeneratedPacketsIndicator.png)
+
+4. Click the **First Display Channel** tab to view the last frame displayed for SO0.
+    > Note that the image resolution matches the 1920x1080 resolution of the TDMS file created for SI0.
+
+    ![Generating and Displaying Multiple Images First Display Channel](../../images/PXIe-148X-Gen-MultipleImages-FirstDisplayChannel.png)
+
+5. Click the **Second Display Channel** tab to view the last frame displayed for SO1.
+    > Note that the image resolution matches the 2880x1280 resolution of the TDMS file created for SI1.
+
+    ![Generating and Displaying Multiple Images Second Display Channel](../../images/PXIe-148X-Gen-MultipleImages-SecondDisplayChannel.png)
+
+6. Select the **Serial Channel** tab and set the **Start Trigger Delay (s)** control value to <font face = "courier new">2</font>.
+   
+   > The settings on the **Serial Channel** tab are now similar to the image below.
+   
+   ![Generating and Displaying Multiple Images Start Trigger Delay](../../images/PXIe-148X-Gen-MultipleImages-StartTriggerDelay.png)
+
+7. Run the VI, wait for the **Waiting for Serializer Setup** indicator to illuminate, and click the **Serializer Setup Complete** control button to start the generation. 
+    > Notice that there is now a 2 second delay before images are generated after clicking the **Serializer Setup Complete** button.
```

Comment:
I agree with this section being here since this control is on the Serial Channel tab, but I think it needs more connecting and explanatory text.  1st point is already covered in the earlier comment I made at the start of this section.

2nd point is a couple recommended but optional explanations (either separate notes or changes to the existing steps to add some explanation)

1. The delay affects the start of image generation that pushed across the wire, and thus the image display is also delayed. Because this section is very focused on display, it's not 100% clear that the delay affects _everything_ not just display.

2. Consider adding a link to gen help document with the explanation of delay and how it interacts with the automatic adjustments that the VI already makes. Something like:

Note: The delay value is added to existing automatic adjustments the VI automatically makes to ensure generation starts at the intended point in time.  Refer to the Start Trigger Delay (s) description in the PXIe-148X Generation GSE Help for additional details.

Reply:
I like this suggestion and I decided to make this a separate section with its own heading.  That gave me a good place to add the necessary description.  I decided for this description to be geared more towards the GSE - how this relates to the Acq GSE and why this control exists.  The Help has more about the low level implementation.  Lastly, I updated the step numbering all subsections in "Setting Serial Channel Configurations" would be continuations from the previous section.  Hopefully that will work in all renderings.

Reply:
Nice.